apps/web/components/booking/AvailableTimes.tsx

```diff
@@ -13,6 +13,7 @@ import { useLocale } from "@lib/hooks/useLocale";
 import { useSlots } from "@lib/hooks/useSlots";
 
 import Loader from "@components/Loader";
+import { useMeQuery } from "@components/Shell";
```

Comment:
This import should be
```
import useMeQuery from "@lib/hooks/useMeQuery";
```
Get the following error otherwise 
<img width="1065" alt="Screenshot 2022-05-25 at 15 45 35" src="https://user-images.githubusercontent.com/6601142/170303989-ad295917-a4e9-4c29-ba64-dd2c3f3baee1.png">